### ISSUE TITLE
Typo in your paper?

### ISSUE THREAD

**Issue by ghost** [wojzaremba/lstm#11]
Hi,

is there a typo in your paper on page 5? It currently reads,

"We train the model for 55 epochs with a learning rate of 1; after 14 epochs we start to reduce the learning rate by a factor of 1.15 after each epoch."

Should it be,

"We train the model for 55 epochs, initially with a learning rate of 1; after 14 epochs we start to reduce the learning rate by a factor of 1.15 after each epoch."

I think that's consistent with lines [259-260](https://github.com/wojzaremba/lstm/blob/master/main.lua#L259) and 32 and 36

Regards,

Aj

Edit - I think there might be a similar typo for the medium model as well?

**Comment by wojzaremba** [comment]
Thank you a lot Ajay. I will update it shortly.

**Comment by ghost** [comment]
Hi Wojciech,

than YOU very much for sharing your code and writing your paper - I think you made a lot of people happy! Using dropout in the way you detailed in your paper has been close to a miracle for the LSTM encoder-decoder system I'm working on :) - its really cool :)))

Kind regards,

Aj

PS - I'm getting some good results using the [Adam optimizer](https://github.com/torch/optim/blob/master/adam.lua), for LSTM encoder-decoders. With gradient clipping, dropout and a high initial learning rate, even a fool like me can train big models :)
